### ISSUE TITLE
Rule block heading contrast failures

### ISSUE THREAD

**Issue by jamesjacobs** [jadu/pulsar#1085]
Standard rule block headings foe "When", "Or" and "Then" do not meet AA colour contrast.

![image](https://user-images.githubusercontent.com/756393/69253482-a04e7c00-0bac-11ea-9f41-83beec9080c4.png)
